### PR TITLE
completed part of translation

### PR DESCRIPTION
**Pull request by trifirew**
context.cson: 80%
menu_darwin.cson: 80%
menu_linux.cson: 80%
menu_win32.cson: 80%
settings.cson: 20%
changed default language to zh-cn

### CODE REVIEW

**Comment by trifirew** [comment]
@liuderchi 抱歉，創建pull request的時候沒有添加label，本人對git的使用還不是很熟練，望多多指教！

**Comment by trifirew** [comment]
I made a mistake by setting the default language to zh-cn. I will set it to English in next commit.

**Comment by trifirew** [comment]
Already changed `package.json` to be the same as the one in `master`.

**Comment by liuderchi** [comment]
@trifirew Thanks in advance for contributing this project ! :+1:

Never mind about labeling, as a owner it's my work to label and organize the PRs.

File: def/zh-cn/menu_darwin.cson [hunk 1]
```diff
@@ -27,197 +27,196 @@ Menu:
...
       "Toggle Comments":
-        value: "Toggle Comments"
+        value: "注释/取消注释"
       Lines:
-        value: "Lines"
+        value: "行操作"
         submenu:
-          Indent:
-            value: "Indent"
+            value: "缩进"
```
Comment (liuderchi) [inline def/zh-cn/menu_darwin.cson]: key `Indent:` is missing. Please fix it.

File: def/zh-cn/menu_darwin.cson [hunk 2]
```diff
@@ -27,197 +27,196 @@ Menu:
...
           Outdent:
-            value: "Outdent"
+            value: "退格"
           "Auto Indent":
-            value: "Auto Indent"
+            value: "自动缩进"
           "Move Line Up":
-            value: "Move Line Up"
+          value: "上移当前行"
```
Comment (liuderchi) [inline def/zh-cn/menu_darwin.cson]: Please keep original the indentation.
`value:` should have 2 more spaces than key `"Move Line Up"`.

Comment (trifirew) [inline def/zh-cn/menu_darwin.cson]: There are fixed now.
I think I should double check the files next time :)

Comment (liuderchi) [inline def/zh-cn/menu_darwin.cson]: Recently a [Travis automatic test](https://travis-ci.org/liuderchi/atom-i18n) has been integrated with Pull Request review process.
If there is any syntax error we can know before merging. :+1:

**Comment by liuderchi** [comment]
Thanks @trifirew for your contribution!
Looks pretty close to 100%

Please notify me when you think it's ready to merge.

**Comment by liuderchi** [comment]
this PR is now deprecated and we move to #36

**Pull request closed without merging**